tillerinobot/src/main/java/tillerino/tillerinobot/lang/Spanish.java

```diff
@@ -17,7 +17,7 @@
 
 	@Override
 	public String unknownBeatmap() {
-		return "Lo siento, no conozco ese mapa. Es muy reciente, muy difícil, unranked o modo de osu! no estándar.";
+		return "Lo siento, no conozco ese mapa. Es muy reciente, muy difícil, no rankeado o modo de osu! no estándar.";
```

Comment:
im not entirely sure this needs to be translated, altho I see other languages do this, there are that dont do this...